### ISSUE TITLE
Error: The getter 'data' isn't defined for the class 'ExifData'.

### ISSUE THREAD

**Issue by Novak-Fenocchio** [chooyan-eng/crop_your_image#89]
Flutter 3.3.10 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 135454af32 (6 weeks ago) • 2022-12-15 07:36:55 -0800
Engine • revision 3316dd8728
Tools • Dart 2.18.6 • DevTools 2.15.0

crop_your_image: ^0.7.3

![CleanShot 2023-01-25 at 10 05 32](https://user-images.githubusercontent.com/68252867/214570861-fe75fa0b-4def-459e-9304-b2415f094c85.png)

**Comment by InspectlyMads** [comment]
Having same issue after upgrading flutter

**Comment by itsAmitGoyani** [comment]
Having same issue. Any update on this?

**Comment by nhatbui23988** [comment]
The easiest way is to try downgrading image-3.3.3 to image-3.2.2
Or upgrade version crop_your_image to 0.7.4
Because in crop_your_image: 0.7.3, file crop.dart has used 'exif.data'
<img width="451" alt="image" src="https://user-images.githubusercontent.com/42292674/216138918-1d22321e-631f-4fd7-8cb9-25b6e96c41ce.png">
which is used in image-3.2.2
<img width="450" alt="image" src="https://user-images.githubusercontent.com/42292674/216139138-16bdaf0e-cf2d-40b6-b983-ebcc0e5c0857.png">
and 'exif.data' is no longer used in image-3.3.3

**Comment by aviavinas** [comment]
This fixes it for me
```
  crop_your_image:
    git: https://github.com/aviavinas/crop_your_image-0.7.3.git
```
I had to manually change image dependency to a specific version

**Comment by chooyan-eng** [comment]
Thank you for your feedback, and I apologize for replying late.

As discussed so far, the dependency on the `image`, package would cause the problem.
I'll try to find out how we can fix this without any workaround.

**Comment by chooyan-eng** [comment]
I've checked and found that it's the problem that was solved on `0.7.4`.
Does anyone still face the same problem after updating `0.7.4`?

**Comment by chooyan-eng** [comment]
Maybe duplicate with #83
